index.html

```diff
@@ -9332,7 +9353,7 @@ <h5>Presentational Role Inheritance</h5>
 					</tr>
 					<tr>
 						<th class="role-childpresentational-head" scope="row">Children Presentational:</th>
-						<td class="role-childpresentational"> </td>
+						<td class="role-childpresentational">True</td>
```

Comment:
And looks like this snuck in as well -- when making PRs, it's good practice to review the diff, because you probably would have caught these things! :)

Reply:
Thanks @spectranaut this has been fixed in [81cd343](https://github.com/chlane/aria/commit/81cd343772f02835c1d27cbf45553726508ee064)

Reply:
Thanks @spectranaut this has been fixed in https://github.com/w3c/aria/pull/1827/commits/0317b3039c121f81949135a8b056015416f52e92